hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionHealthProblem.java

```diff
@@ -0,0 +1,116 @@
+package pl.allegro.tech.hermes.api;
+
+public enum SubscriptionHealthProblem {
```

Comment:
there is an ongoing trend that we put stuff into production and then we have to tweak stuff from configuration because we don't know good values upfront, extracting to configuration helps to shorten feedback loop